src/components/news/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Fragment, useCallback } from "react"
+import { useToggle } from "react-use"
+import { Layer, Flex } from "@netdata/netdata-ui"
+import Header from "./header"
+import Item from "./item"
+import { Container } from "./container"
+
+const News = ({ children, items = [], onCloseClick }) => {
```

Comment:
~~onCloseClick~~ -> onClose

Reply:
I insist with `onCloseClick` 😛 . Let's keep it consistent with the documentation helper also!